### PR TITLE
Update free-courses-ar.md

### PR DESCRIPTION
**Pull request by mostafa-amine**
## What does this PR do?
Add Arabic resource to learn the basics of laravel in Arabic 👍

### Description
I have added one of the best Arabic courses to understand the basics of laravel
### Why is this valuable (or not)?
Because it will enable any Arab person to understand the basics of the Laravel framework

### How do we know it's really free?
Because it is on YouTube

### For book lists, is it a book? For course lists, is it a course? etc.
Yes, its a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ar.md [hunk 1]
```diff
@@ -381,7 +381,8 @@
 * [دورة انشاء برنامج فواتير php- laravel&rlm;](https://www.youtube.com/playlist?list=PLftLUHfDSiZ7pKXkpGCoZATm5rF6msj5A) - Mora Soft
 * [دورة Laravel 6/7/8 PHP&rlm;](https://www.youtube.com/playlist?list=PLMYF6NkLrdN8V2JKIMxqMsZNPsgUj3WOK) - Muhammed Essa
 * [كورس اساسيات لارافيل 9](https://www.youtube.com/playlist?list=PLftLUHfDSiZ4GfPZxaFDsA7ejUzD7SpWa) - Mora Soft
-* [مشروع لارافل متجر الكتروني متعدد التجار واللغات متكامل - تطبيقات لارافل عمليه](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP7DCb-NamG2tt7uQUfxP2va) - Professional Code
+* [لارافيل للمبتدئين](https://www.youtube.com/playlist?list=PLWCBAKY7-4buQazvDjeZhjLl54UqbF3lM) - Ahmed Abd El Ftah
+*  [مشروع لارافل متجر الكتروني متعدد التجار واللغات متكامل - تطبيقات لارافل عمليه](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP7DCb-NamG2tt7uQUfxP2va) - Professional Code
```
Comment (eshellman) [inline courses/free-courses-ar.md:385]: is there an extra space here?

Comment (mostafa-amine) [reply courses/free-courses-ar.md:385]: i don't think so.

Comment (eshellman) [reply courses/free-courses-ar.md:385]: there are 2 spaces after `*` in line 385, all the other entries have just on e space

Comment (mostafa-amine) [reply courses/free-courses-ar.md]: Yes I solved the problem.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**